CommonTools/RecoAlgos/plugins/JetConstituentSelector.cc

```diff
@@ -97,7 +97,9 @@ class JetConstituentSelector : public edm::EDFilter {
 typedef JetConstituentSelector<reco::PFJet> PFJetConstituentSelector;
 typedef JetConstituentSelector<pat::Jet, std::vector< edm::FwdPtr<pat::PackedCandidate> > > PatJetConstituentSelector;
 typedef JetConstituentSelector<reco::PFJet, std::vector< edm::FwdPtr<pat::PackedCandidate> > > MiniAODJetConstituentSelector;
+typedef JetConstituentSelector<reco::PFJet, std::vector< edm::FwdPtr<reco::PFCandidate> > > MyMiniAODJetConstituentSelector;
```

Comment:
This is the same as a PFJetConstituentSelector. It doesn't actually function correctly with packed candidates as input, which is why I needed to make the others.